### ISSUE TITLE
Use Person Type database data for all uses of PERSON_TYPE

### ISSUE THREAD

**Issue by andrew-rist-tw** [PublicDataWorks/complaint-manager#256]
~Blocked by #160~

### Business Context:
As part of migrating away from instance-files we will have moved PERSON_TYPEs into the database and created a GET handler for them. Now we want to make sure all imports of the old PERSON_TYPE constant are removed and replace with database queries (server side) or service calls (client side) to get that data.

**As an** org adopting CM
**I want** person types in the database
**So that** I don't have to fill in person types in an instance-files project

### In Scope
1. Update database where there are references to the person type (either the key or description) and replace them with foreign keys to the person type table
2. Move PersonType class into complaint-manager

### Out of Scope
1. Editing Person Types in Admin Portal
2.
3.

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: 
Given
When
Then

2. Scenario: 
Given
When
Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**

**Comment by mariaDelSignore** [comment]
tables/views affected by change

- cases_officers
- civilians
- sortable_cases_view
